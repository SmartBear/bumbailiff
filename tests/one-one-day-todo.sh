yesterday=$(date -v -1d) # OSX/BSD only!
mkdir -p tmp/git/one-zero-day-todo
cd tmp/git/one-zero-day-todo
git init --quiet .
git commit --quiet -m "Initial commit" --allow-empty --date "$yesterday"
echo "// TODO: 1-day old TODO" > file.js
git add .
git commit --quiet -m "Commit" --date "$yesterday"
bumbailiff
echo $?
