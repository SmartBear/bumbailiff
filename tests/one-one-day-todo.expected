[1mfile.js[0m
[0;32m1 days old[0;37m <a-commit-ref> (Matt Wynne <a-timestamp> 1)[0m
TODO: 1-day old TODO

ðŸŒ± [0;32m1 TODO-days accumulated (max allowance: 14 ðŸ’µ).[0m
0
